debian/changelog

```diff
@@ -1,3 +1,96 @@
+gtk+3.0 (3.24.11-1endless1) UNRELEASED; urgency=medium
```

Comment:
```suggestion
gtk+3.0 (3.24.11-1endless0) UNRELEASED; urgency=medium
```

Reply:
1endless1 is fine (AFAIK).

Reply:
I've been using 1endless1 so far, I don't know how I got this idea but 1endless0 would imply to me that we didn't change anything from the Debian package — I'd propose to keep 1endless1, let me know if you still disagree though and I will reconsider it.

Reply:
On the other hand, our instructions in https://phabricator.endlessm.com/w/howto/rebase_debian_packages/ do explicitly agree with you. I will make this change.